WebRoot/templates/share/Dialogs.template

```diff
@@ -400,21 +400,33 @@
 </template>
 
 <template id="share.Dialogs#ZmNewTagDialog">
-	<table role="presentation" class='ZPropertySheet' cellspacing='6'>
+	<table role="presentation" class='ZmDialogPropertySheet' cellspacing='6'>
```

Comment:
do we need cellspacing here?

Reply:
No, it's not needed. I'll remove it. Thanks for pointing it out.

Thanks